ddl.dic

```diff
@@ -2320,10 +2320,13 @@ save_units.code
 
     _definition.id                '_units.code'
     _definition.class             Attribute
-    _definition.update            2021-03-01
+    _definition.update            2023-06-10
     _description.text
 ;
     A code which identifies the units of measurement.
+
+    'unspecified' is used only for List, Array, Matrix, or Table data types
+    where the constituent values have differing units.
```

Comment:
```suggestion
    The 'unspecified' code should only be used in cases when the units cannot
    be properly expressed in DDLm. A typical example of such situation is a
    List data item with constituent values that may have differing units.
```

Reply:
I suggest we generalise it a bit more.

Should we maybe also use this enumeration value as a placeholder in situations when the units are determined using dREL calculations (e.g. `_refln.a_calc`)? Software that properly handles dREL would simply override this values whereas less capable software could at least properly discern that the `_units.code` attribute was not accidentally left out.

Reply:
Can't disagree with that. (but also not having (knowingly) worked with software that does proper dREL),